config/config_types.go

```diff
@@ -214,6 +215,12 @@ type Exchange struct {
 	WebsocketURL                     *string              `json:"websocketUrl,omitempty"`
 }
 
+// WebsocketSubscriptions holds the websocket subscriptions for an exchange
+type WebsocketSubscriptions struct {
+	Authenticated   map[string][]string `json:"authenticatedSubscriptions"`
```

Comment:
Would be cool to have a struct with enabled bool as well so we can on off whatever is supported.

Reply:
Yeah great stuff. I like your ideas and they _shouldn't_ blow out too much more than this. I didn't want to try and do a catch all, because I'd rather improvements happen when needed and this not taking too long. 

Will look at the enabled/disabled style with predefined toggles with maybe a string slice for ones outside of that.

 I'll see how I go before looking at wrapper funcs though to minimise the amount of changes